build-system/server/typescript-compile.js

```diff
@@ -55,7 +65,23 @@ function buildNewServer() {
   }
 }
 
+/**
+ * @return {string}
+ */
+function getTypeCheckCmd() {
+  switch (process.platform) {
+    case 'win32':
+      return `node .\\node_modules\\typescript\\lib\\tsc.js --noEmit -p ${SERVER_TRANSFORM_PATH.split(
```

Comment:
`npx`?

Reply:
Does that work? I can just sub out the rest with npx? I'll try

Reply:
Of course it duz. Fixed.